specification/quota/resource-manager/Microsoft.Quota/preview/2025-03-15-preview/examples/GroupQuotaLimitsRequests/PatchGroupQuotaLimitsRequests-Compute.json

```
@@ -0,0 +1,97 @@
+    "resourceProviderName": "Microsoft.Compute",
+    "location": "westus",
+    "api-version": "2025-03-15-preview",
+    "groupQuotaRequest": {
+      "properties": {
+        "value": [
+          {
+            "properties": {
+              "limit": 110,
+              "resourceName": "standardddv4family",
+              "comment": "Contoso requires more quota."
+            }
+          },
+          {
+            "properties": {
+              "limit": 110,
+              "resourceName": "standardav2family",
+              "comment": "Contoso requires more quota."
+            }
+          }
+        ]
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/providers/Microsoft.Management/managementGroups/E7EC67B3-7657-4966-BFFC-41EFD36BAA09/providers/Microsoft.Quota/groupQuotas/groupquota1/resourceProviders/Microsoft.Compute/groupQuotaLimits/westus",
+        "name": "westus",
+        "type": "Microsoft.Quota/groupQuotas/groupQuotaLimits",
+        "properties": {
+          "provisioningState": "Succeeded",
+          "value": [
+            {
+              "properties": {
+                "limit": 100,
+                "resourceName": "standardddv4family",
+                "name": {
+                  "value": "standardddv4family",
+                  "localizedValue": "standard DDv4 Family vCPUs"
+                },
+                "unit": "count",
+                "availableLimit": 50,
+                "allocatedToSubscriptions": {
+                  "value": [
+                    {
+                      "subscriptionId": "00000000-0000-0000-0000-000000000000",
+                      "quotaAllocated": 20
+                    },
+                    {
+                      "subscriptionId": "A000000-0000-0000-0000-000000000000",
+                      "quotaAllocated": 30
+                    }
+                  ]
+                }
+              }
+            },
+            {
+              "properties": {
+                "limit": 100,
+                "resourceName": "standardav2family",
+                "name": {
+                  "value": "standardav2family",
+                  "localizedValue": "Standard AV2 Family vCPUs"
+                },
+                "unit": "count",
+                "availableLimit": 80,
+                "allocatedToSubscriptions": {
+                  "value": [
+                    {
+                      "subscriptionId": "00000000-0000-0000-0000-000000000000",
+                      "quotaAllocated": 20
+                    }
+                  ]
+                }
+              }
+            }
+          ],
+          "nextLink": "https://yourLinkHere.com"
+        }
+      }
+    },
+    "202": {
+      "headers": {
+        "location": "https://contoso.com/operationstatus",
+        "Azure-AsyncOperation": "https://contoso.com/operationstatus",
+        "Operation-Location": "https://contoso.com/operationstatus",
+        "retry-after": 30
```

```suggestion
        "retry-after": "30"
```

I believe values of `retry-after` must be strings, not integers.

---

thank you! is it required that all the "retry-after" type must be strings besides the one place in the alert? Can I fix them together in this PR?

---

You will probably need to fix all examples.  If your examples are also broken in main, you'll want to fix them in a separate PR, and keep this PR just for your new API version.

---

gotcha, I created another PR: [SubscriptionQuotaAllocation](https://github.com/Azure/azure-rest-api-specs/pull/33283) to fix the type first. But it's still the same error. do you mind suggesting what do I miss there?

---

I'm not sure what all you are trying to do in 33283.  My first thought was you were just updating your examples, but you are also updating your spec, which itself might be a real breaking change.

---

oh but in the spec, the type is defined as int64. if I only change the example to string, wouldn't that be a mismatch?

---

no, as explained here: https://github.com/Azure/autorest/issues/4988#issuecomment-2220564282

---

Hi @mikeharder, do you mind reviewing again for me? Other checks passed but there is a lintdiff one, which has the [Test Ignore] tag so I assume can be ignored?

---

Ignore any checks prefixed with `[TEST-IGNORE]`.  I do not review PRs, you will need to wait until your turn in the ARM review queue.